pkg/cmd/server/origin/master_config.go

```diff
@@ -641,6 +641,12 @@ func (c *MasterConfig) OriginNamespaceControllerClients() (*osclient.Client, *kc
 	return c.PrivilegedLoopbackOpenShiftClient, c.PrivilegedLoopbackKubernetesClient
 }
 
+// UnidlingControllerClients returns the unidling controller clients
+func (c *MasterConfig) UnidlingControllerClients() (*osclient.Client, *kclient.Client) {
+	// TODO(directxman12): make this use service account clients
```

Comment:
Will be required for merge - follow the pattern for the other ones (need to create role and SA).